subset/bacnet/Dockerfile.test_bacext

```diff
@@ -0,0 +1,22 @@
+FROM daq/aardvark:latest as bacnet_build
+
+# Do this alone first so it can be re-used by other build files.
+RUN $AG update && $AG install openjdk-8-jre
+
+RUN $AG update && $AG install openjdk-8-jdk git
+
+#This refer to a specific github commit on the repo
+ENV BACHASH=6b4493f5d9b4e2640ebb9dcc659e5e05fda61b33
```

Comment:
hash should be the same in all cases.

Reply:
I need more clarification on this as I'm writing all bacnet tests in my bacnet4j library forked from you. Every time I commit a new change I need to update the env in my dockerfile and test_***. It'd like to know your advice on this for future reference.